src/widgets/Object.vue

```diff
@@ -19,42 +18,15 @@
 						:tab="myTab"
 						:activeTab="activeTab"
 						:depth="depth"
-						:key="propName"
-						v-if="shouldCreateProp(propName)" />
-						<b-btn @click="addProp(propName)" v-else-if="isPostponedProp(propName)">add {{ propName }}</b-btn>
+						:key="propName" />
+					<!-- commented out make it clear that this kind of functionality used to exist -->
+					<!-- <b-btn @click="addProp(propName)" v-else-if="isPostponedProp(propName)">add {{ propName }}</b-btn>-->
```

Comment:
lets not leave commented out code, it does not bring any value, git history will have the change and we have the issue id as well. make sure that the commit message will mention that this was removed.

Reply:
In this case I thought that it is not easy to know that such functionality would be useful or possible so I left the code there. If it would be a commit message only it would be hard for the next guy who has to possibly re-implement it to know that such thing ever existed. On the other hand I'm not even sure if it ever worked or there is use case for this.

Reply:
having commented out code makes the codebase rot too fast. There is the git history for each file, which will store the history. All unused code lines are a possible issue in the future, as the code will change around it, and there is no guarantee that it will work with the future code

Reply:
Okey, removing it.